exist-docker/pom.xml

```diff
@@ -213,22 +213,23 @@
                   <verbose>true</verbose>
                     <images>
                         <image>
-                            <name>existdb/existdb:%v</name>
+                            <name>existdb/existdb</name>
                             <alias>exist</alias>
                             <build>
                                 <tags>
+                                    <tag>%v</tag>
                                     <tag>${docker.tag}</tag>
                                 </tags>
                                 <dockerFile>${project.build.outputDirectory}/Dockerfile</dockerFile>
                                 <contextDir>${assemble.dir}</contextDir>
                             </build>
                         </image>
                         <image>
-                            <name>existdb/existdb:%v-DEBUG</name>
+                            <name>existdb/existdb:debug</name>
                             <alias>exist-debug</alias>
                             <build>
                                 <tags>
-                                    <tag>${docker.debug.tag}</tag>
+                                    <tag>%v-debug</tag>
```

Comment:
This is not the same as before. The name was previously `x.x.x-DEBUG` or `x.x.x-SNAPSHOT-DEBUG`.

Reply:
the names were  `existdb/existdb:debug` or `existdb/existdb:X.X.X-DEBUG` with `%v` also creating `existdb/existdb:latest` for semVers ending in `-SNAPSHOT` which was causing problems. Since that should no be created using the Debug Dockerfile

Reply:
> with %v also creating existdb/existdb:latest
Yes, but that was in a separate`<image>` block and so should not impact this.

Reply:
Can you point me to the docs that speak of this isolation? What we saw on ci here, matches my reading of their docs on multiple images.

Reply:
If this change fixes a problem maybe it is better to change the tooling to now look for the new tag?
Alternatively, is it OK and sufficient to just uppercase from `%v-debug` to `%v-DEBUG`?